README.md

```diff
@@ -19,16 +19,12 @@ width="240"/></a>
  * Dedicated buttons for zoom in/out to show the map sharp
  * Show user position
  * Tracking user position (long click on show position)
-  
-## Hidden functions
-
  * Open a location using the app: click on a link with coordinated or address
- * Switch between matric and imperial units in settings
+ * Switch between metric and imperial units in settings
  
-## Requested functions, ideas and plans
+## Requested functions, ideas & plans
 
  * Add other maps - ok
- * Offline - probably yes
```

Comment:
I would suggest to keep offline function in the ideas.

It would be great if you can research offline function of osmdroid library and propose some improvements for OSMV.